cfgov/jinja2/v1/_lib/django_calendar_event_processor.py

```diff
@@ -1,46 +0,0 @@
-import sys
```

Comment:
i see these have been removed, are they no longer necessary? i thought they were being used for the sheer_indexing